<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/logo.png</normaloff>../icons/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    background-color: rgb(211,211,211);
    color: rgb(88, 88, 90);
    selection-background-color: rgb(21, 165, 137);
    selection-color: rgb(245, 244, 244);
    
	font: 16pt
}

QLabel
{
  background-color: rgb(211,211,211);
}

QLabel#error_label
{
   color: rgb(220, 130, 30);
}

QTabBar::tab:top:selected {
    background-color: rgb(21, 165, 137);
    color: rgb(88, 88, 90);
}

</string>
  </property>
  <property name="iconSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tab_widget">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>500</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QMenu" name="main_menu">
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="action_connect"/>
    <addaction name="action_close_session"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="config_menu">
    <property name="title">
     <string>Configure</string>
    </property>
    <addaction name="action_check_configuration"/>
    <addaction name="action_add_configuration"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Views and Plugins</string>
    </property>
   </widget>
   <addaction name="main_menu"/>
   <addaction name="config_menu"/>
   <addaction name="menuPlugins"/>
  </widget>
  <action name="action_close_session">
   <property name="text">
    <string>Close Connection</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit iBridges</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionSaveConfig">
   <property name="text">
    <string>Save configuration</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="action_connect">
   <property name="text">
    <string>Connect to iRODS</string>
   </property>
  </action>
  <action name="action_check_configuration">
   <property name="text">
    <string>Check Configuration</string>
   </property>
  </action>
  <action name="action_add_configuration">
   <property name="text">
    <string>Add Configuration</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
